.golangci.yml

```diff
@@ -55,3 +55,7 @@ issues:
 
   - path: pkg/apis/kubeone
     text: "type name will be used as kubeone.KubeOneCluster by other packages"
+
+  - path: test/e2e
+    text: "cyclomatic complexity 33 of func `TestClusterConformance` is high"
```

Comment:
I propose that we ignore this for now, but get back to it once we start working on #839.